doc/api/next_api_changes/behavior/28375-MP.rst

```diff
@@ -0,0 +1,5 @@
+``transforms.AffineDeltaTransform`` updates correctly on axis limit changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before this change, transform sub-graphs with ``AffineDeltaTransform`` did not update correctly.
+This PR ensures that changes to axis limits are passed through correctly to the underlying transform.
```

Comment:
Wording is not really optimal (considering that this will be part of a changelog).  Also the reference to axes limits may be a bit too specific (AffineDeltaTransform does not refer to transData/transLimits in itself).

Reply:
My lens on this was a bit hyper-specific. I am happy to go with any suggestion you may have on this.

Reply:
At the very least, it's "when the child transform changes", not specifically axis limits (as that's only the case for the example in #28364.)

Reply:
@QuLogic Thanks for the suggestion. Done.